core/commands/storage/session.go

```diff
@@ -345,9 +345,6 @@ func (ss *FileContracts) SetFileHash(fileHash cidlib.Cid) {
 }
 
 func (ss *FileContracts) GetFileHash() cidlib.Cid {
-	ss.Lock()
```

Comment:
Note: in refactor, remove all Get* and Set* routines if they need no locks or complex logic.

Reply:
okay